src/Microsoft.OData.Client/EntityDescriptor.cs

```diff
@@ -248,7 +248,7 @@ internal set
         public String ServerTypeName { get; internal set; }
 
         /// <summary>Returns a collection of links that are the relationships in which the entity participates.</summary>
-        /// <returns>A <see cref="T:System.Collections.ObjectModel.ReadOnlyCollection`1" /> of <see cref="T:Microsoft.OData.Client.LinkInfo" /> objects that represents links in which the entity participates.</returns>
+        /// <returns>A <see cref="System.Collections.ObjectModel.ReadOnlyCollection`1" /> of <see cref="Microsoft.OData.Client.LinkInfo" /> objects that represents links in which the entity participates.</returns>

```

Comment:
I didn't resolve ALL warnings. I will split this across multiple pull requests, as otherwise it will get too big to review. There will still be warnings left after this pull request is completed. Either way, your comment is definitely valid, I added the change.